src/modal/modal.stache

```diff
@@ -6,15 +6,21 @@
     tabindex="-1"
     role="dialog"
     aria-hidden="true"
-    class="modal fade"
+    class="modal {{#if(fullscreen)}}maximized{{else}}fade in{{/if}}"
     aria-labelledby="pageModalLbl"
   >
 
     <div class="modal-dialog {{#if(modalContent.textlongFieldVM.field.name)}}modal-lg{{/if}}">
       <div class="modal-content">
         {{#if(modalContent)}}
         <div class="modal-header">
-          <button type="button" class="close" data-dismiss="modal" aria-label="Close" aui-action="close" tabindex="1">
+          {{#if(allowFullscreen)}}
+            <button type="button" class="fullscreen" on:click="toggleFullscreen()" aria-label="Fullscreen" tabindex="1">
```

Comment:
Does the `tabindex` need to be on the button? I think it should be focusable without it.

I would consider adding a `title` attribute to this too, so mouse users can hover over it and see what it does.

Reply:
The tab index is already set to 1 elsewhere in the modal so adding it to the new button keeps everything tabbing how you'd expect. Didn't want to change legacy stuff too much for a last min addition.

title attribute makes sense, a11y audit probably would want it on the close button too, yeah? I'll add it to both

Reply:
Yeah, that’d be helpful.

I don’t think we should have the `tabindex`, but I’d say leave it for consistency and if I discover problems with it, I’ll submit another PR.